ocaml/xapi/xapi_cluster.ml

```diff
@@ -196,5 +196,5 @@ let pool_destroy ~__context ~self =
 
 let pool_resync ~__context ~self =
   List.iter
-    (fun host -> Xapi_cluster_host.resync_host ~__context ~host)
+    (fun host -> log_and_ignore_exn (fun () -> Xapi_cluster_host.resync_host ~__context ~host))
```

Comment:
This loop is good, but might be useful to check at the end whether the resync succeeded and report back an error to the user. i.e. check that each host has a matching cluster host (if clustering is enabled), and that all cluster hosts are enabled.
Otherwise this API call always succeeds even if something failed.